web/intents/intents_test.go

```diff
@@ -134,7 +135,7 @@ func TestMain(m *testing.M) {
 		fmt.Println(err)
 		os.Exit(1)
 	}
-	appPerms, err = permission.CreateWebappSet(ins, webapp.Slug(), webapp.Permissions())
+	appPerms, err = permission.CreateWebappSet(ins, webapp.Slug(), webapp.Permissions(), metadata.New())
```

Comment:
We have a `slug` here. We should use it for the metadata of the permission doc, no?